azurelinuxagent/common/osutil/gaia.py

```diff
@@ -151,8 +151,6 @@ def eject_dvd(self, chk_err=True):
         logger.warn('eject is not supported on GAiA')
 
     def mount(self, device, mount_point, option=None, chk_err=True):
-        logger.info('mount {0} {1} {2}', device, mount_point, " ".join(option))
```

Comment:
Why was this removed?

Reply:
I had added this logging statement in the other run_command PR, and noticed that it should be removed in this one.

Reply:
Thanks, I didn't realize you had introduced it, I thought it was there from the beginning. Makes sense then!